dev-libs/libtypec/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <longdescription>
+    “libtypec” is aimed to provide a generic interface abstracting all
+    platform complexity for user space to develop tools for efficient
+    USB-C port management. The library can also enable development of
+    diagnostic and debug tools to debug system issues around USB-C/USB
+    PD topology. 
+  </longdescription>
+  <!-- maintainer-needed -->
```

Comment:
We can't add new packages as maintainer-needed: https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602.

Normally, we tell people to put new packages in GURU or another repository unless/until a developer wants it in ::gentoo, or some package needs it/or it's a long-standing contributor. You fall into the latter so I'm happy to give it a chance. See https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml if you're willing to adopt it. Thanks!

Reply:
Sounds good to me!